### PR TITLE
Fixed bug with updating allocated pool VM

### PR DESCRIPTION
**Pull request by bond95**
Bug: After allocating new pool VM, it doesn't appear in VM list.
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/593

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-web-ui/611)

### CODE REVIEW

File: src/sagas.js [hunk 1]
```diff
@@ -457,22 +457,26 @@ function* startProgress ({ vmId, poolId, name }) {
...
-  const getSingle = vmId ? getSingleVm : getSinglePool
   const actionInProgress = vmId ? vmActionInProgress : poolActionInProgress
-
-  const params = vmId ? { vmId } : { poolId }
   if (result && result.status === 'complete') {
     // do not call 'end of in progress' if successful,
     // since UI will be updated by refresh
     yield delay(5 * 1000)
-    yield fetchSingle(getSingle(params))
+    yield getSingleInstance({ result, poolId })
```
Comment (mareklibra) [inline src/sagas.js]: Can we pass `vmId` instead of the `result` here?

Comment (bond95) [reply src/sagas.js]: Done.

**Comment by bond95** [comment]
@mareklibra Done.

File: src/sagas.js [hunk 2]
```diff
@@ -457,22 +457,26 @@ function* startProgress ({ vmId, poolId, name }) {
   }
 }
 
+function* getSingleInstance ({ vmId, poolId }) {
+  yield fetchSingleVm(getSingleVm({ vmId }))
```
Comment (jniederm) [inline src/sagas.js]: We can probably use concurrent fetching:

```js
const fetches = [ fetchSingleVm(getSingleVm({ vmId })) ]
if (poolId) {
  fetches.push(fetchSinglePool(getSinglePool({ poolId })))
}
yield all(fetches)
```

**Pull request merged**